iothub/service/samples/PnpServiceSamples/TemperatureController/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using PnpHelpers;
+
+namespace TemperatureController
+{
+    public class Program
+    {
+        // Get connection string and device id inputs
+        private static readonly string s_hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
+        private static readonly string s_deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
+
+        // These are values as defined in DTMI used for device client sample with component.
+        // DTDL interface used: https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json
+        private const string ComponentName = "thermostat1";
+
+        // Writable property to update
+        private const string PropertyName = "targetTemperature";
+        private const double PropertyValue = 60;
+
+        // Method on a given component
```

Comment:
Method on `Thermostat` component.

Reply:
It's ok too keep it generic for comments

Reply:
This is a sample targeting a particular model, why would we want to keep it generic?

Reply:
component is thermostat1/ thermostat2 right? so whichever user specified